main/tests/cypress/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": false,
+    "singleQuote": true
```

Comment:
We should get these reviewed by the team before making large-scale changes based on them. 

I think our existing Javascript mostly uses 2 space indenting and trailing semicolons.

Reply:
This is the standard prettier configuration for cypress based project. But if the team has other preferences, I can make those edits accordingly.

Reply:
I think all of our Javascript should conform to a common style, whether it's test related or not. I don't have a strong preference of 2 vs 4 spaces, but if we're going to switch to 4 spaces, we should do it all at once and put tooling in place to enforce it. Of course, this will introduce a lot of noise across the entire repo, so we should make sure it's worth the cost.